airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -188,7 +188,8 @@ public DestinationConnection getDestinationConnection(final UUID destinationId)
     return persistence.getConfig(ConfigSchema.DESTINATION_CONNECTION, destinationId.toString(), DestinationConnection.class);
   }
 
-  public void writeDestinationConnection(final DestinationConnection destinationConnection) throws JsonValidationException, IOException {
+  public void writeDestinationConnection(final DestinationConnection destinationConnection, final ConnectorSpecification connectorSpecification)
+      throws JsonValidationException, IOException {
```

Comment:
Can we add the validation sanity check, similar to the source connection? Otherwise it's hard to verify that we're getting the correct value passed here in tests.

Reply:
Done